docs/main/quickstart-guide/quickstart.md

```diff
@@ -44,87 +39,82 @@ The OpenEBS workflow fits nicely into the reconcilation pattern introduced by Ku
 
-As a Kubernetes cluster administrator or Platform SREs you will have to decide which deployment strategy works best for you - either use an hyperconverged mode where Stateful applications and storage volumes are co-located or run Stateful applications and storage on different pools of nodes. 
+As a Kubernetes cluster administrator or Platform SREs, you will have to decide which deployment strategy works best for you - either use an hyperconverged mode where Stateful applications and storage volumes are co-located or run Stateful applications and storage on different pools of nodes. 
 
 For installing OpenEBS, you Kubernetes cluster should meet the following:
 - Kubernetes 1.18 or newer is recommended. 
 - Based on the selected data engine, the nodes should be prepared with additional packages like:
-  - Installing the ext4, xfs, nfs, lvm, zfs or iscsi, nvme packages.
+  - Installing the ext4, xfs, nfs, lvm, zfs, nvme packages.
   - Prepare the devices for use by data engines like - making sure there are no the filesystem installed or by creating an LVM volume group or ZFS Pool or partition the drives if required. 
 - Based on whether you are using a upstream Kubernetes cluster or using a managed Kubernetes cluster like AKS, Rancher, OpenShift, GKE, there may be additional steps required. 
 
-Please read through the relevant section of the [pre-requisites](/user-guides/prerequisites) for your Kubernetes platform, Operating System of the worker nodes.
+Read through the relevant section of the [pre-requisites](../user-guides/local-engine-user-guide/prerequisites.mdx) for your Kubernetes platform, Operating System of the worker nodes.
 
-- [Ubuntu](/user-guides/prerequisites#ubuntu)
-- [RHEL](/user-guides/prerequisites#rhel)
-- [CentOS](/user-guides/prerequisites#centos)
-- [OpenShift](/user-guides/prerequisites#openshift)
-- [Rancher](/user-guides/prerequisites#rancher)
-- [ICP](/user-guides/prerequisites#icp)
-- [EKS](/user-guides/prerequisites#eks)
-- [GKE](/user-guides/prerequisites#gke)
-- [AKS](/user-guides/prerequisites#aks)
-- [Digital Ocean](/user-guides/prerequisites#do)
-- [Konvoy](/user-guides/prerequisites#konvoy)
+- [Ubuntu](../user-guides/local-engine-user-guide/prerequisites.mdx)
+- [RHEL](../user-guides/local-engine-user-guide/prerequisites.mdx)
+- [CentOS](../user-guides/local-engine-user-guide/prerequisites.mdx)
+- [OpenShift](../user-guides/local-engine-user-guide/prerequisites.mdx)
+- [Rancher](../user-guides/local-engine-user-guide/prerequisites.mdx)
+- [ICP](../user-guides/local-engine-user-guide/prerequisites.mdx)
+- [EKS](../user-guides/local-engine-user-guide/prerequisites.mdx)
+- [GKE](../user-guides/local-engine-user-guide/prerequisites.mdx)
+- [AKS](../user-guides/local-engine-user-guide/prerequisites.mdx)
+- [Digital Ocean](../user-guides/local-engine-user-guide/prerequisites.mdx)
+- [Konvoy](../user-guides/local-engine-user-guide/prerequisites.mdx)
```

Comment:
Did we miss the positional reference with #?

Reply:
No Vishnu. We didn't miss the positional references. All these sections were talking about **iSCSI** which is not applicable now and those sections are removed now. Hence, the positional references are not made.

Also, now the "Prerequisites" page won't be there in the new structure as we're planning to have separate Prerequisites for each engine.